src/scene/batching/batch-group.js

```diff
@@ -2,17 +2,59 @@ import { LAYERID_WORLD } from '../constants.js';
 
 /**
  * Holds mesh batching settings and a unique id. Created via {@link BatchManager#addGroup}.
- *
- * @property {boolean} dynamic Whether objects within this batch group should support transforming
- * at runtime.
- * @property {number} maxAabbSize Maximum size of any dimension of a bounding box around batched
- * objects. {@link BatchManager#prepare} will split objects into local groups based on this size.
- * @property {number} id Unique id. Can be assigned to model and element components.
- * @property {string} name Name of the group.
- * @property {number[]} [layers] Layer ID array. Default is [{@link LAYERID_WORLD}]. The whole
- * batch group will belong to these layers. Layers of source models will be ignored.
  */
 class BatchGroup {
+    /** @private */
+    _ui = false;
+
+    /** @private */
+    _sprite = false;
+
+    /** @private */
+    _obj = {
+        model: [],
+        element: [],
+        sprite: [],
+        render: []
+    };
+
+    /**
+     * Unique id. Can be assigned to model and element components.
```

Comment:
please mention the render component here too

Reply:
Fixed!